staging/src/k8s.io/client-go/discovery/cached/memory/memcache.go

```diff
@@ -103,7 +112,10 @@ func (d *memCacheClient) ServerResourcesForGroupVersion(groupVersion string) (*m
 	if cachedVal.err != nil && isTransientError(cachedVal.err) {
 		r, err := d.serverResourcesForGroupVersion(groupVersion)
 		if err != nil {
-			utilruntime.HandleError(fmt.Errorf("couldn't get resource list for %v: %v", groupVersion, err))
+			// Do not log errors for "empty response", which is a common response for metrics.
```

Comment:
should we log empty errors at V(3) verbosity like diskcache?

Reply:
Done